src/Compilers/Core/Portable/WellKnownTypes.cs

```diff
@@ -260,6 +260,8 @@ internal enum WellKnownType
         System_ValueTuple_T7,
         System_ValueTuple_TRest,
 
+        System_Runtime_CompilerServices_TupleElementNamesAttribute,
+
         Available,
         Last = Available - 1,
```

Comment:
We should consider removing `WellKnownTypes.Last`. Other than the use below (which requires updating each time an item is added to `WellKnownTypes`) `Last` is only used for range checking, and those uses can use `Available` instead (perhaps renamed to `Next`): `type >= WellKnownType.First && type < WellKnownType.Next`.